alacritty/src/display/mod.rs

```diff
@@ -622,6 +619,10 @@ impl Display {
         // Resize renderer.
         let physical = PhysicalSize::new(self.size_info.width() as _, self.size_info.height() as _);
         self.window.resize(physical);
+
+        // Ensure we're modifying the correct OpenGL context.
+        self.window.make_current();
```

Comment:
Have you checked that there are no opengl calls above this line? When I experimented with this originally, I also moved the code related to the glyph cache below this line.

Reply:
Hm, yeah, you're right. there's a case when loading fonts, so we should move it past it as well.

Reply:
I've moved it above other calls, so should be fine.